pypowsybl/network_modification.py

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2022, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+import pypowsybl._pypowsybl as _pp
+from numpy.typing import ArrayLike as _ArrayLike
+from pandas import DataFrame as _DataFrame
+
+from pypowsybl.network import Network
+from pypowsybl.report import Reporter as _Reporter
+from pypowsybl.utils.dataframes import _adapt_df_or_kwargs, _create_c_dataframe
+
+
+def create_new_feeder_bay(n: Network, busbar_section_id: str, injection_position_order: int,
```

Comment:
would it be better in network module ?

Reply:
I thought about putting it in this module because the network module is pretty big and there will be a lot of functions added for modifications. But I can move it.